prow/config.yaml

```diff
@@ -709,35 +709,25 @@ presubmits:
     trigger: "(?m)^/test( all| pull-cluster-registry-bazel),?(\\s+|$)"
     labels:
       preset-service-account: true
+      preset-bazel-scratch-dir: true
```

Comment:
this mounts an emptyDir for the local $HOME/.cache/bazel equivilant and points bazel at this. it should be faster than working in the overlayFS and is consistent with what we had before *except* that it isn't persisted. the env var below / image switch enable the remote cache.

Reply:
Very nice, thank you!

I think I'll probably send out a follow-up PR to update some of our other tests that depend on bazel as well, if possible: some (perhaps all) of our other tests use bazel to build code generators or other build tools, and it would be nice to speed them up as well.